editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -169,34 +171,27 @@ void VersionControlEditorPlugin::_set_credentials() {
 }
 
 bool VersionControlEditorPlugin::_load_plugin(String p_name) {
-	String path = ScriptServer::get_global_class_path(p_name);
-	Ref<Script> script = ResourceLoader::load(path);
+	Object *extension_instance = ClassDB::instantiate(p_name);
+	ERR_FAIL_NULL_V_MSG(extension_instance, false, TTR("Received a nullptr VCS extension instance during construction."));
 
-	ERR_FAIL_COND_V_MSG(!script.is_valid(), false, "VCS Plugin path is invalid");
-
-	EditorVCSInterface *vcs_interface = memnew(EditorVCSInterface);
-	ScriptInstance *plugin_script_instance = script->instance_create(vcs_interface);
-
-	ERR_FAIL_COND_V_MSG(!plugin_script_instance, false, "Failed to create plugin script instance.");
-
-	// The plugin is attached as a script to the VCS interface singleton as a proxy end-point
-	vcs_interface->set_script_instance(plugin_script_instance);
-
-	EditorVCSInterface::set_singleton(vcs_interface);
+	EditorVCSInterface *vcs_plugin = Object::cast_to<EditorVCSInterface>(extension_instance);
+	ERR_FAIL_NULL_V_MSG(vcs_plugin, false, vformat(TTR("Could not cast VCS extension instance to %s."), EditorVCSInterface::get_class_static()));
```

Comment:
I won't comment on all of them but you should remove all `TTR()` in `ERR_FAIL.*` macros.

As you can see before this PR there's almost no code translating `ERR_FAIL` errors:
```
$ rg ERR_FAIL.*TTR
editor/editor_vcs_interface.cpp
173:                    ERR_FAIL_MSG(TTR("Couldn't create .gitignore in project path."));
180:                    ERR_FAIL_MSG(TTR("Couldn't create .gitattributes in project path."));

editor/import/resource_importer_imagefont.cpp
115:    ERR_FAIL_COND_V_MSG(err != OK, ERR_FILE_CANT_READ, TTR("Can't load font texture:") + " \"" + p_source_file + "\".");
```